src/payment/payment-methods/cybersource/service.js

```diff
@@ -18,15 +23,15 @@ App.ensureConfig([
  * 'hits' then we must not perform the transaction.
  */
 export async function sdnCheck(basketId, firstName, lastName, city, country) {
-  const { data } = await App.apiClient.post(
-    `${App.config.ECOMMERCE_BASE_URL}/api/v2/sdn/search/`,
+  const { data } = await getAuthenticatedHttpClient().post(
+    `${getConfig().ECOMMERCE_BASE_URL}/api/v2/sdn/search/`,
     {
       name: `${firstName} ${lastName}`,
       city,
       country,
     },
   ).catch((error) => {
-    logApiClientError(error, {
+    logError(error, {
       messagePrefix: 'SDN Check Error',
```

Comment:
Another messagePrefix - partially I'm just worried that the fact we're not actually using it as a prefix will break some reporting on the other side

Reply:
Going to stop commenting on individual messagePrefix instances

Reply:
This was stuff I'd added. I don't there's any reporting hooked up to search for these error messages. AFAIK